### ISSUE TITLE
NVDA reads out table grid's cell postion - Can this be avoided?

### ISSUE THREAD

**Issue by noobards** [nvaccess/nvda#5947]
Hi,
I've set up a fiddle here [**JSFIDDLE**](https://jsfiddle.net/j49gz3g3)

While navigating between dates using the keyboard arrow keys, NVDA reads out

> Friday, May 12 2016 row 3
> Thursday, May 4 2016 column 5

whereas the expected output is

> Friday, May 12 2016
> Thursday, May 4 2016

Is there a way in which the reading out of the cell position (row 3, column 5) be avoided? Or is this a standard behavior with tables involving data?

This is happening in all major browsers (viz IE, FF, Chrome)

PS: Also attached a screenshot for reference.

![nvda](https://cloud.githubusercontent.com/assets/7556369/15248714/48d34c62-1939-11e6-8518-666e9284b1eb.jpg)

**Comment by josephsl** [comment]
Hi, Try disabling “report table cell coordinates” option from Document Formatting dialog (note that NVDA does not have ability to provide domain0specific profiles at this time). Thanks.

**Comment by noobards** [comment]
Thanks  **josephsl** That indeed did the trick.
